package.json

```diff
@@ -22,6 +22,24 @@
         2,
         "always",
         "Signed-off-by"
+      ],
+      "scope-enum": [
+        2,
```

Comment:
Shouldn't this be a warning instead of an error?   I mean it says "optional" in the above CONTRIBUTING.md explainer.

Reply:
In the scope-enum section in https://commitlint.js.org/reference/rules.html has a note

> This rule always passes if no scopes are provided in the message or the value > is an empty array.

It seems that the scope is always optional.

Reply:
> In the scope-enum section in https://commitlint.js.org/reference/rules.html has a note
> 
> > This rule always passes if no scopes are provided in the message or the value > is an empty array.
> 
> It seems that the scope is always optional.

Sounds good to me.  I tried to figure out always|never|optional but missed the scope-enum detail (and there is no optional).